sdk/ai/ai-inference-rest/README.md

```diff
@@ -14,9 +14,9 @@ Key links:
 ## Getting started
 
 ```javascript
-import ModelClient, { isUnexpected } from "@azure-rest/ai-inference";
+import createClient, { isUnexpected } from "@azure-rest/ai-inference";
```

Comment:
If we are supposed to use this method instead of the class constructor, shouldn't this method have a more descriptive name like `createModelClient`?

Reply:
/cc @joheredi @qiaozha @MaryGao

Reply:
I believe since it's a default export, you can use any names, `createModelClient` should work too.

Reply:
[Updated ](https://github.com/Azure/azure-sdk-for-js/pull/32838/commits/4408540581969783e7c02a737c627ece9cb13bc5) to use ModelClient, to match RLC conventions in other Azure JS SDKs.